### ISSUE TITLE
Fix ReadTheDocs deprecation issue

### ISSUE THREAD

**Issue by nickreich** [hubverse-org/hubDocs#151]
More detail here: https://about.readthedocs.com/blog/2024/07/addons-by-default/

**Comment by zkamvar** [comment]
I've been trying this out and from what I can tell, there's not much that needs to be changed... maybe upgrading myst, but I've been able to get a rtd site going with the following requirements: https://github.com/zkamvar/effective-barnacle/blob/main/docs/requirements.txt

That being said, I am having a pickle of a time because I keep getting a "MIMEtype" warning from the `_static/` folder 🤷🏽, but I think it's not really a big deal.

**Comment by zkamvar** [comment]
I have confirmed that it's not a big deal for the site by building from a fork. I also went ahead and tested with a more up-to-date versions of sphinx and myst and it's nice because the text is a lot clearer. 

The only challenge is the tables are zebra-striped by default and there is a bug where the table classes are cast to lowercase: https://github.com/executablebooks/MyST-Parser/issues/960, but those are easy to work around.

https://dubhawks.readthedocs.io/en/latest/user-guide/sample-output-type.html#four-submissions-differing-by-compound-modeling-task
